cernopendata/modules/fixtures/data/docs/delphi-guide-docker/delphi-guide-docker.md

```diff
@@ -1,80 +1,72 @@
+$ xhost + local:docker
+$ docker run --privileged --rm -it -e DISPLAY --network host -v /tmp/.X11-unix:/tmp/.X11-unix -v ~/.Xauthority:/home/delphi/.Xauthority --user delphi gitlab-registry.cern.ch/delphi/deployment/delphi/al9_64 /bin/bash -l
 ```
 
-This command will download the Alma9 based container and create a login shell for the DELPHI user. This image comes with support for EOS: when launched on a system which supports fuse file systems, the DELPHI data will be available inside the container beneath the path `/eos/opendata/delphi`. Note that due to the --rm option, the container will be destroyed when you exit it.
+This command will download the Alma9 based container and create a login shell for the DELPHI user.
+
+This image comes with support for `EOS`: when launched on a system which supports fuse file systems, the DELPHI data will be available inside the container beneath the path `/eos/opendata/delphi`. Note that due to the --rm option, the container will be destroyed when you exit it, and all the work done inside the container will be lost.
 
 <p><center><img src="/static/docs/delphi-guide-docker/delphi-container-start.png" width="60%"></center></p>
 
 ## Contents of the container image
 The image ships with the following modules:
 
-* CERNLIB: This is the community CERNLIB version
-* [dstana](/record/80502): dstana is the analysis framework of DELPHI. For historical reasons there are 3 different versions available. The default one and the one which should be used is the pro version. Only switch to a non default version is you know what you are doing.
-* simana: Simana is the simulation and reconstruction framework of DELPHI. Note that it differs for each year. For the year 2000 there are 2 different versions: va0e and va0s. The former is valid for the first year, the latter for the second part of the year when a part of the TPC went offline.
-* [delgra](/record/80503): delgra is the event display of delphi.
+* CERNLIB: non-free [community version](https://cernlib.web.cern.ch)
+* [dstana](/record/80502) is the analysis framework of DELPHI. For historical reasons there are 3 different versions available. The default one and the one which should be used is the pro version. Only switch to a non default version is you know what you are doing.
+* `simana`  contains the simulation and reconstruction framework of DELPHI. Note that it differs for each year. For the year 2000 there are 2 different versions: va0e and va0s. The former is valid for the first year, the latter for the second part of the year when a part of the `TPC` went offline.
+* [delgra](/record/80503) is the event display of delphi.
 
-The software stack itself can be found in /delphi on the container.
+The software stack itself can be found in `/delphi` on the container.
```

Comment:
can be found in the `/delphi` directory on the container.

(to make it clearer)